### PR TITLE
[WIP] - Adding spiff enabled bosh manifest and cloud formation template to set up the concourse system in the AWS environment.

### PR DESCRIPTION
**Pull request by adrianwebb**

### CODE REVIEW

File: concourse/concourse.yml [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+# replace with an AZ from your account
+availability_zone: &az (( meta.az ))
+
+# replace all addresses with your VPC range
+#
+# e.g. X.X.0.2 -> 10.0.0.2
+networks:
+  - name: concourse
+    type: manual
```
Comment (dlapiduz) [inline concourse/concourse.yml:65]: Do you know if we can make this network dynamic? I've run into issues before with manual networks...

**Comment by dlapiduz** [comment]
@adrianwebb I don't see a new subnet for the concourse stuff... don't we need it?

**Comment by adrianwebb** [comment]
@dlapiduz It should be able to use an existing internal service subnet.  That's pretty much why I left all the network configurations to configure in the secrets file.  I don't know which one we might use yet though (we have a lot of subnets in our AWS environment).

File: concourse/concourse.yml (hunk 1 above)
Comment (adrianwebb) [inline concourse/concourse.yml:65]: We might be able to.  I have not experimented with the dynamic network configuration yet though.  I can test it out in my personal AWS environment and report my results back.

**Comment by dlapiduz** [comment]
@adrianwebb it wouldn't be a terrible idea to create a new one just for this... it would help if we have to do permissions

**Comment by adrianwebb** [comment]
@dlapiduz Ok.  Will do.  I'll get that added to the CFT today.  I'll need to coordinate with you on an acceptable subnet CIDR.

**Comment by adrianwebb** [comment]
@dlapiduz How do the changes look to the concourse cloudformation template?  I added subnet creation and routing configuration.

**Comment by adrianwebb** [comment]
@dlapiduz can you take a look at this pull request and merge if it looks good?

**Pull request merged**